themes/meraki/sections/desktop-slider.liquid

```diff
@@ -286,10 +286,16 @@
       "settings": [
         {
           "type": "image_picker",
-          "id": "image",
-          "label": "Image",
+          "id": "desktop_image",
+          "label": "🖥 Desktop Image",
           "info": "To ensure your images look great and load quickly, use a size of \"1920px x 1080px\" (16:9 aspect ratio)."
         },
+        {
+          "type": "image_picker",
+          "id": "mobile_image",
+          "label": "📱 Mobile Image",
+          "info": "To ensure your images look great and load quickly, use a size of \"500 x 550px\" (16:9 aspect ratio)."
+        },
```

Comment:
the dimensions `500 x 550px` do not correspond to a `16:9` aspect ratio.
Try `500 x 500px`- `1:1`